### PR TITLE
Update upload.html

### PR DESCRIPTION
**Pull request by Manvi07**
A basic front-end for uploading cif-files. Addresses #5

### CODE REVIEW

File: tools-barebone/webservice/templates/upload.html [hunk 1]
```diff
@@ -1 +1,44 @@
...
+                <li class="active"><a href="/">Home</a></li>
+                <li><a href="upload">Upload file</a></li>
+                <li><a href="view">View files</a></li>
+              </ul>
+            </div>
+          </nav>
+
+          <div class="jumbotron text-center">
+              <h1> Validate your cif files </h1>
+              <form action="./compute/validate/" id="uploadfile" method="POST">
```
Comment (rishi-s8) [inline tools-barebone/webservice/templates/upload.html]: This should be ```action="/validate/"```

**Comment by rishi-s8** [comment]
Also it is not working because I suppose the key being sent is ```file``` instead of ```cif``` to the back-end, resulting in a Bad Request.

**Comment by Manvi07** [comment]
changes made :)

**Comment by rishi-s8** [comment]
Did you check if it is working?

**Comment by rishi-s8** [comment]
@Manvi07, still a Bad Request.

**Comment by Manvi07** [comment]
Let me see.

File: tools-barebone/webservice/templates/upload.html [hunk 2]
```diff
@@ -1 +1,44 @@
...
+                <li class="active"><a href="/">Home</a></li>
+                <li><a href="upload">Upload file</a></li>
+                <li><a href="view">View files</a></li>
+              </ul>
+            </div>
+          </nav>
+
+          <div class="jumbotron text-center">
+              <h1> Validate your cif files </h1>
+              <form action="/compute/validate/" id="cif" method="POST">
```
Comment (rishi-s8) [inline tools-barebone/webservice/templates/upload.html]: This should be ```<form action="/compute/validate/" id="upload-form" method="POST" enctype="multipart/form-data">```

**Comment by rishi-s8** [comment]
Working. :) I don't like the titles of those options though. :( 
This is good work. Complete and then we'll merge.

**Comment by Varunvaruns9** [comment]
This can be closed in favor of #17 .

**Comment by rishi-s8** [comment]
Closing this PR.

**Pull request closed without merging**